masterlist.yaml

```
@@ -6311,6 +6316,11 @@ plugins:
         crc: 0xF0CE4E14
         util: '[FO4Edit v4.0.1](https://www.nexusmods.com/fallout4/mods/2737)'
         nav: 1
+      # version: 1.01 after 2nd pass of quick clean
+      - <<: *reqManualFix
+        crc: 0x9429AA85
+        util: '[FO4Edit v4.0.4](https://www.nexusmods.com/fallout4/mods/2737)'
+        nav: 1
```

You can replace the existing cleaning data.

```suggestion
        crc: 0x9429AA85
        util: '[FO4Edit v4.0.4](https://www.nexusmods.com/fallout4/mods/2737)'
        nav: 1
```